cves/CVE-2016-7401.yml

```diff
@@ -66,30 +68,24 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: d1bc980db1c0fffd6d60677e62f70beadb9fe64a
-  note:
+  note: | 
+    The fix was to parse the cookies in a different manner checking
+    for any invalid strings. Tests were added and updated.
 - commit: 6118ab7d0676f0d622278e5be215f14fb5410b6a
-  note:
+  note: | 
+    The fix was to parse the cookies in a different manner checking
+    for any invalid strings. Tests were added and updated.
 vccs:
 - commit: b4066d7d2149782cd694b98d1a6ed5c7ee885f55
-  note: This VCC was discovered automatically via archeogit.
-- commit: c159374a33caee14d9903de58a9f4fe879e5741c
-  note: This VCC was discovered automatically via archeogit.
+  note: Moved cookie code out of __init__.py.
 - commit: 42c31f6bf036efd93c0311bc1f524b1553c20489
-  note: This VCC was discovered automatically via archeogit.
-- commit: '088f7170779b98547d9862603387afee77b964a8'
-  note: This VCC was discovered automatically via archeogit.
-- commit: 42b5e4feeacf7cfa57867bf9fd5a6046de8c1cd3
-  note: This VCC was discovered automatically via archeogit.
-- commit: 38408f8007eae21b9f1cbbcc7f86d4b2042ff86a
-  note: This VCC was discovered automatically via archeogit.
-- commit: f7969b0920c403118656f6bfec58d6454d79ef1a
-  note: This VCC was discovered automatically via archeogit.
-- commit: d63ee2097f4d0f0094e8e472de2236558e98a348
-  note: This VCC was discovered automatically via archeogit.
-- commit: 0ed7d155635da9f79d4dd67e4889087d3673c6da
-  note: This VCC was discovered automatically via archeogit.
-- commit: 75536fef1ff56894e35dad221a4215ebe3917673
-  note: This VCC was discovered automatically via archeogit.
+  note: Moved cookie class.
+- commit: 088f7170779b98547d9862603387afee77b964a8
+  note: Added cookie class.
+- commit: 2659429df46d0d79ad7a93d50d0ccdab45b55b53
+  note: |
+    The original VCC was part of modifying how cookies were passed
+    as a bug prevented them from being passed through load.
```

Comment:
Changing this sentence to something like "The original VCC contained changes that modified the way in which cookies were being passed, as a bug was preventing them from being passed through load" would make it a bit easier to understand.

Reply:
Added clarification.